src/constraints/investment.jl

```diff
@@ -13,7 +13,11 @@ function add_investment_constraints!(graph, Ai, Fi, assets_investment, flows_inv
         if graph[a].capacity > 0 && !ismissing(graph[a].investment_limit)
             JuMP.set_upper_bound(
                 assets_investment[a],
-                graph[a].investment_limit / graph[a].capacity,
+                if graph[a].investment_integer
+                    floor(graph[a].investment_limit / graph[a].capacity)
+                else
+                    graph[a].investment_limit / graph[a].capacity
+                end,
```

Comment:
Add something like
```julia
value = graph[a].investment_limit / graph[a].capacity
if graph[a].investment_integer
    value = floor(value)
end
```
outside the function call and just use `value` instead. `bound_value` might be a better name, but since it is inside the `if`, it doesn't matter a lot.